seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -2479,6 +2482,42 @@ def test_multi_project_all_samples_all_inheritance_get_es_variant_gene_counts(se
 
         self.assertCachedResults(results_model, {'gene_aggs': gene_counts, 'total_results': 5})
 
+    @urllib3_responses.activate
+    def test_all_samples_any_affected_get_es_variant_gene_counts(self):
+        setup_responses()
+        search_model = VariantSearch.objects.create(search={
+            'annotations': {'frameshift': ['frameshift_variant']}, 'inheritance': {'mode': 'any_affected'},
+        })
+        results_model = VariantSearchResults.objects.create(variant_search=search_model)
+        results_model.families.set(Family.objects.filter(project__guid='R0001_1kg'))
```

Comment:
Could the data for VariantSearchResults be put into the fixture data?

Reply:
It isn't in the other 29 tests in the file, so that would be an anti-pattern